movieland-web/src/main/java/com/gromoks/movieland/web/interceptor/LoggerInterceptor.java

```diff
@@ -0,0 +1,61 @@
+package com.gromoks.movieland.web.interceptor;
+
+import com.gromoks.movieland.service.security.AuthenticationService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.UUID;
+
+public class LoggerInterceptor extends HandlerInterceptorAdapter {
+
+    private static Logger log = LoggerFactory.getLogger(LoggerInterceptor.class);
+    private static final String GUEST = "guest";
+
+    @Autowired
+    private AuthenticationService authenticationService;
```

Comment:
not required - you use static methods

Reply:
It's not static methods but make sense to do static